### PR TITLE
Reduce change detection cycles

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

File: scroller/package.json [hunk 1]
```diff
@@ -6,8 +6,5 @@
     "@angular/common": ">=12.0.0",
     "@angular/core": ">=12.0.0",
     "vscroll": "^1.5.4"
-  },
-  "dependencies": {
-    "tslib": "^2.3.1"
```
Comment (arturovt) [inline scroller/package.json:11]: Removed because it's added automatically by `ng-packagr` when the library is being built.

**Comment by arturovt** [comment]
@dhilt I was having troubles to fix unit tests locally, there're lots of them tho. Could you help me fixing them?

**Comment by dhilt** [comment]
@arturovt This looks very interesting, thanks for PR! Give me a few days for review.

**Comment by dhilt** [comment]
@arturovt ngx-ui-scroll v3.1.0 is released, it contains your updates

Thanks for your contribution, it seems super relevant and even led to some changes in the core lib.

**Comment by arturovt** [comment]
Thank you @dhilt !

**Pull request merged**